client/src/pages/transaction/tokens-create.tsx

```diff
@@ -0,0 +1,40 @@
+import { Code, Table, Tbody, Td, Tr } from "@liftedinit/ui";
+import { Link } from "react-router-dom";
+
+export function tokensCreate(data: any) {
+  const summary = data.argument.summary;
+  return {
+    Token: (
+      <Code as={Link} to={`/addresses/${data.argument.owner}`} color="brand.teal.500">
+        {summary.ticker}
+      </Code>
+    ),
+    Holders: (
+      <Table>
+        <Tbody>
+          {Object.entries(data.argument.holders).map(([address, amount]) => (
+            <Tr>
+              <Td >
+                {(
+                  parseInt(amount as string) /
+                  10 ** summary.precision
+                ).toLocaleString()}{" "}
+                {summary.ticker}
+              </Td>
+              <Td>
+                <Code
+                  as={Link}
+                  to={`/addresses/${address}`}
+                  color="brand.teal.500"
+                >
+                  {address}
+                </Code>
+              </Td>
```

Comment:
I would swap the two columns, i.e., `address: amount TOKEN`

Reply:
![image](https://github.com/liftedinit/talib/assets/3028410/2d9576c3-b20d-4e3b-8451-c6821200d7e5)

Reply:
Not a fan of the additional horizontal bar.

Reply:
Okay, fixed on both create token table and mint-burn tables 
![image](https://github.com/liftedinit/talib/assets/3028410/8d45c002-a597-4a76-9ea7-2b42a69d436c)